nss-day-cohort-51/nss-day-cohort-51.github.io
# updated youtube links on dev cards

# Description
Updated broken youtube links on Morgan and Devery's modal
## Type of change

- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update

# Testing Instructions
Run npm start from branch
click on view more for Devery and Morgan
See youtube video properly displayed

# Checklist:

- [] I have commented my code, particularly in hard-to-understand areas
